app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class='my-3' >
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field border-bottom" style='height: 50px;'>
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email", class: 'form-control border-0 rounded-0 w-100 h-100' %>
+    </div>
+
+    <div class="field" style='height: 50px;'>
+      <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password", class: 'form-control  border-0 rounded-0 w-100 h-100'  %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions">
+      <%= f.submit "Log in", class: 'btn btn-primary p-1 rounded-lg text-white' %>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] I can sign in, but can't log in. I would advise you to look at it, Authorize the user to log in.
![1](https://user-images.githubusercontent.com/63972670/221362822-d447483a-213d-4c01-bb43-0f5b42826695.JPG)

Reply:
Hi @OLIPLICHE 

Hope you are well.

I think you can't log in because you have to confirm this email shown in the terminal after registration, that's because I add `:confirmable` to User Authentication.

Reply:
- Tks for your feedback. As you know, We sign up before you log in. We use the same email and password to log in(That is where the issue is coming). I checked your `devise` of course confirmable is there.